static/app/bootstrap/index.tsx

```diff
@@ -34,7 +34,68 @@ async function bootWithHydration() {
   }
   window.__initialData = data;
 
-  return bootApplication(data);
+  bootApplication(data);
+  preloadOrganizationData(data);
+
+  return data;
+}
+
+function promiseRequest(url: string): Promise<any> {
+  return new Promise(function (resolve, reject) {
+    const xhr = new XMLHttpRequest();
```

Comment:
We're already using `fetch` in our bundle and it's not polyfilled. Any reason we can't keep using fetch?

Reply:
Ah I see yo just brought this up

Reply:
I just forklifted the old code. I can make another pass to use fetch() as it will yield smaller a bundle size, which could be relevant as this code is early in the bundle list.